cli/tsc.rs

```diff
@@ -65,6 +65,19 @@ fn get_maybe_hash(
   }
 }
 
+fn hash_data_url(
+  specifier: &ModuleSpecifier,
+  media_type: &MediaType,
+) -> String {
+  assert_eq!(
+    specifier.as_url().scheme(),
+    "data",
+    "Specifier must be a data: specifier."
+  );
+  let hash = crate::checksum::gen(&[specifier.as_url().path().as_bytes()]);
```

Comment:
Not really, because the content will be the same, irrespective of what the query string is.  So for internal caching purposes a `data:application/javascript,whatever` is the same as `data:application/javascript,whatever?query`, so the same hash is desired.

Reply:
It's contingent of whether what's after `?` is included in the content. I'll run some tests on Chrome and get back to you.

Reply:
I just added `import "data:text/javascript;,console.log('Hello?world')";` on top of a JS module and Chrome printed `Hello?world` in the console. This means that `data:text/javascript;,console.log('Hello?world')` and `data:text/javascript;,console.log('Hello?universe')` would be hashed the same, although they are different URLs.